metal-nexus.spec

```diff
@@ -50,13 +50,14 @@ Provides: pit-nexus
 %define sonatype_nexus3_image artifactory.algol60.net/csm-docker/stable/docker.io/sonatype/nexus3:%{sonatype_nexus3_tag}
 %define sonatype_nexus3_file  sonatype-nexus3-%{sonatype_nexus3_tag}.tar
 
-%define cray_nexus_setup_tag   0.10.1
+%define cray_nexus_setup_tag   0.11.0
```

Comment:
Version `0.11.0` of `cray-setup-nexus` makes requests to `/v1` Nexus API where previous `/beta` was accessed: https://github.com/Cray-HPE/nexus-setup/pull/29. The `/beta` is not supported anymore by Nexus 3.67.1.